src/test/regress/sql/multi_size_queries.sql

```diff
@@ -20,6 +20,8 @@ SELECT citus_total_relation_size('non_distributed_table');
 DROP TABLE non_distributed_table;
 
 -- Tests on distributed table with replication factor > 1
+VACUUM (FULL) lineitem_hash_part;
```

Comment:
Do we have this vacuum so that the size will be consistent?

Reply:
Yes